lib/services/lists.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const _isEqual = require('lodash/isEqual'),
+  db = require('./db');
+
+/**
+ * Appends an entry to a list and returns the updated list.
+ * @param {string} uri
+ * @param {Array<Object>|Object} data
+ * @returns {Array<Object>}
+ */
+async function addToList(uri, data) {
+  // do a db call for the list
+  const list = await db.get(uri),
+    added = list.concat(data);
```

Comment:
It might be a good idea to make sure the item being added to the list isn't already there.

Reply:
supposedly we want to allow dupes... which makes add / delete a little difficult. decided to allow dupes to be added, and deletion will remove all duplicates.

Reply:
yeah Chris lists aren't sets as far as clay is concerned i.e. they don't have semantics for duplicates, so dupes should be allowed

Reply:
How will this work with components like `tags` which has a `count`?

Reply:
is `count` used anywhere ?  afaik it's only tracked, and I don't think it's tracked very well.  At least I remember it being inaccurate.

If we care about `count` then a generic clay list isn't a good structure for this since there's no good way to update it while taking into account concurrent modifications.  A new structure with an identifier would be necessary.